workshops/2019-10-29_ODSC-WEST/max-fashion-mnist-wrapping-solution/README.md

```diff
@@ -276,6 +276,31 @@ All you need to start wrapping your model is pre-processing, post-processing and
    
 6. Finally, assign the output from the post-processing step to the appropriate response field in `api/predict.py` to link the processed model output to the API.
 
+   - Comment the below line by adding `#` before the start of the line. This is because we are already extracting
```

Comment:
(1) The indentation is not right for the 'Before the change' and 'After the change'
(2) I think it's a little long and confusing, can we just say:

Replace the original code:
```
label_preds = [{'label_id': p[0], 'label': p[1], 'probability': p[2]} for p in [x for x in preds]]
result['predictions'] = label_preds
```

with 

```
result['predictions'] = preds
```

Reply:
I just wanted to convey why we are removing. Fine, I will tell this when explaining. 
Updating now.

Reply:
@splovyt Updated. Is that ok?